### ISSUE TITLE
[BUG] Could not get the historical exchange rate from <date>

### ISSUE THREAD

**Issue by JCalHij** [ghostfolio/ghostfolio#2089]
**Bug Description**

Attempting to introduce a transaction in a currency which does not correspond to the security's base currency, Ghostfolio tries to acquire the exchange rate information for said date. If too old, it will not allow the user to save the transaction.

**To Reproduce**

At least another currency must be set up on the system. In my case, EUR was introduced and set as default account currency.

1. Add Activity
2. Choose USD symbol (e.g. MSFT)
3. Set date to a couple years back (e.g., 6th April 2021)
4. Change unit price to different currency.
5. Error occurs.

**Expected behavior**

Either no error occurs (data is properly gathered from wherever it comes from), or the user is prompted to introduce their own conversion rate.

**Screenshots**

![ghostfolio_exchange_rate_issue](https://github.com/ghostfolio/ghostfolio/assets/38001598/e0d69fdb-9704-4fe6-8a54-de5fac0fc79e)

**Environment**
- Self-hosted
- Ghostfolio Version: 1.282.0
- Browser: Google Chrome
- OS: Debian 12

**Comment by JCalHij** [comment]
I don't know exactly how currency exchange rates are handled. If they are first checked against the posgres database, then it might be a good idea to pre-populate it on creation with some currency data. This link (https://www.ecb.europa.eu/stats/policy_and_exchange_rates/euro_reference_exchange_rates/html/index.en.html) provides a time series of a lot of currency exchange rates with respect to the euro, since 1999. Might be helpful.

**Comment by JCalHij** [comment]
I followed these steps:

1. Opened _Admin Account_ -> _Market Data_. USDEUR symbol appears on the list. 
2. Clicked on list item and a pop up appeared. 
3. Clicked on the top right menu, then _Gather Historical Data_.
4. Closed pop up.
5. Opened it again. Data is now available up to 2013.
6. Tested with new activity from 2021 in USD, using EUR as currency. No problem.

Thanks for the info @dtslvr . Issue can be closed.
